scripts/build_docker_builder.sh

```diff
@@ -15,11 +15,26 @@ if [ $PRIVATE_ECR == "true" ]
 then
   aws ecr get-login-password --region $REGION | docker login -u AWS --password-stdin $ECR_URL
   aws ecr describe-repositories --repository-names $REPO_NAME-builder --region $REGION || aws ecr create-repository --repository-name $REPO_NAME-builder --region $REGION
+  aws ecr describe-repositories --repository-names $REPO_NAME-root --region $REGION || aws ecr create-repository --repository-name $REPO_NAME-root --region $REGION
 else
   aws ecr-public get-login-password --region us-east-1 | docker login -u AWS --password-stdin $ECR_URL
   aws ecr-public describe-repositories --repository-names $REPO_NAME-builder --region us-east-1 || aws ecr-public create-repository --repository-name $REPO_NAME-builder --region us-east-1
+  aws ecr-public describe-repositories --repository-names $REPO_NAME-root --region us-east-1 || aws ecr-public create-repository --repository-name $REPO_NAME-root --region us-east-1
 fi
 
+mkdir -p ./project-package-jsons/projects/default-project
+mkdir -p packages/projects/projects
+cp packages/projects/default-project/package.json ./project-package-jsons/projects/default-project
+find packages/projects/projects/ -name package.json -exec bash -c 'mkdir -p ./project-package-jsons/$(dirname $1) && cp $1 ./project-package-jsons/$(dirname $1)' - '{}' \;
+
+docker buildx build \
+    --load \
+    --build-arg NODE_ENV=$NODE_ENV \
+    -t $ECR_URL/$REPO_NAME-root:${TAG} \
+    -t $ECR_URL/$REPO_NAME-root:latest_$STAGE \
+    -f dockerfiles/root/Dockerfile-root .
+docker push --all-tags $ECR_URL/$REPO_NAME-root
```

Comment:
We should be able to just add the --push option to docker buildx build, and use --cache-to and --cache-from of type gha, like in the other buildx build commands in this file.

Reply:
This has been changed accordingly.